src/api/api.ts

```diff
@@ -1,4 +1,4 @@
-import { authInterceptor as insightsAuthInterceptor } from '@redhat-cloud-services/frontend-components-utilities/files/interceptors';
+import { authInterceptor as insightsAuthInterceptor } from '@redhat-cloud-services/frontend-components-utilities/esm/interceptors';
```

Comment:
It is supposed to be `@redhat-cloud-services/frontend-components-utilities/interceptors`
See here: https://github.com/RedHatInsights/frontend-components/tree/master/packages/utils#import-path-change

Reply:
Thanks, updated